docs/11-circleci/01-getting-started.mdx

```diff
@@ -0,0 +1,149 @@
+handle the activation process if you have all necessary environment variables set in the job's
+context. If you proceed to [set up your workflow](#setting-up-a-workflow) without doing so, your job
+will fail. For more details, head to the [Activation](/docs/circleci/activation) section.
+
+## Setting up a workflow
+
+Create a `.circleci/config.yml` file in the root of your repository and follow these steps:
+
+1. Import the [Unity Orb](https://circleci.com/developer/orbs/orb/game-ci/unity) and fix it to a
+   `minor` version (e.g. `1.16`) to avoid receiving updates with breaking changes:
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+```
+
+2. Include a [build job](/docs/circleci/build) in your workflow and modify the
+   [parameters](/docs/circleci/build#parameters) to accommodate your project needs:
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          name: 'build-linux64-il2cpp'
+          step-name: 'Build StandaloneLinux64'
+          unity-license-var-name: 'UNITY_ENCODED_LICENSE'
+          unity-username-var-name: 'UNITY_USERNAME'
+          unity-password-var-name: 'UNITY_PASSWORD'
+          executor:
+            name: 'unity/ubuntu'
+            target_platform: 'linux-il2cpp'
+            editor_version: '2021.3.1f1'
+            resource_class: 'large'
+          project-path: 'Unity2D-Demo-Game-CI-CD/src'
+          build-target: StandaloneLinux64
+          compress: true
+          context: unity
+```
+
+3. Optionally, include a [test job](/docs/circleci/test) in your workflow:
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  test-and-build-unity-project:
+    jobs:
+      - unity/test:
+          name: 'test-linux'
+          step-name: 'Check if tests run and results are uploaded'
+          unity-license-var-name: 'UNITY_ENCODED_LICENSE'
+          unity-username-var-name: 'UNITY_USERNAME'
+          unity-password-var-name: 'UNITY_PASSWORD'
+          executor:
+            name: 'unity/ubuntu'
+            target_platform: 'linux-il2cpp'
+            editor_version: '2021.3.1f1'
+            resource_class: 'medium'
+          project-path: 'Unity2D-Demo-Game-CI-CD/src'
+          test-platform: 'playmode'
+          context: unity
+      - unity/build: ...
+```
+
+You can add it as a [requirement](https://circleci.com/docs/configuration-reference#requires) in the
+build job to prevent it from running in case of failure:
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  test-and-build-unity-project:
+    jobs:
+      - unity/test:
+          name: 'test-linux'
+          ...
+      - unity/build:
+          ...
+          requires:
+            - test-linux
+```
+
+4. Commit and push the new workflow.
+
+## Downloading the artifacts
+
+1. Navigate to your project in the CircleCI Dashboard.
```

Comment:
```suggestion
1. Navigate to your project in the CircleCI [web app](https://app.circleci.com/).
```
In the CircleCI docs we refer to it as the "web app," and I think it would be preferable to do the same here as well.